crates/ra_hir/src/ty/tests.rs

```diff
@@ -3007,7 +3007,7 @@ impl<T, U> Into<U> for T where U: From<T> {}
 fn test() { S2.into()<|>; }
 "#,
     );
-    assert_eq!(t, "S1");
+    assert_eq!(t, "{unknown}");
```

Comment:
hm yeah, these were relying on the fact that there's only one `U: From<S2>`, weren't they...

Reply:
shouldn't we remove/change this test then? It seems like now it might not be testing the thing it was originally written for. I guess, this is better be handled in a separate PR?